Show claimed context when already claimed

alfetopito opened the pull request:
# Summary

Continues work started by Nenad adding a bit more context when claim is already claimed

![Screen Shot 2022-02-09 at 13 57 44](https://user-images.githubusercontent.com/43217/153298247-39e498ad-e2fc-479d-a3d7-886a6723fe7e.png)

  # To Test

1. Claim everything with one account
* You should see the amount of vCOW claimed for that account
* You should see an updated body and button text as well
2. Check other states (account with no claims and account with claims)
* Should remain as they where

Commit:
Showing claimed vCOW balance amount in summary when already claimed

Commit:
Updating body message when user already claimed

Commit:
Update button text when already claimed

Commit:
Added isClaimed to `ClaimCommonTypes`

alfetopito commented:
Feels a bit redundant (`already claimed` text both on the banner and the body). Any suggestions to improve the text?

elena-zh commented:
Hey @alfetopito , nice changes!

> Feels a bit redundant (already claimed text both on the banner and the body). Any suggestions to improve the text?

I'd suggest to show only one message about 'already claimed' , and show it in the warning at the top of the form. Maybe, I'd add a network indicator into the warning
![claimed](https://user-images.githubusercontent.com/70885163/153402563-61e2c8e5-af53-4d06-aeeb-96b74cc4a91f.jpg)

Also, 1 tiny nitpick: we do not replace 'Connect wallet' button with a disabled button when check an info for an account being not connected. 
Maybe we should follow the same logic when displaying 'already claim' screen? WDYT?
![change button](https://user-images.githubusercontent.com/70885163/153402833-b2516939-9e5f-4bed-9f2c-482dda47aa9d.jpg)
![connect a wallet](https://user-images.githubusercontent.com/70885163/153402839-6eb222da-9600-4823-9790-27c6457099fb.jpg)

nenadV91 commented on src/custom/pages/Claim/ClaimSummary.tsx:
```diff
@@ -7,23 +7,31 @@ import { ClaimSummary as ClaimSummaryWrapper, ClaimSummaryTitle, ClaimTotal } fr
 import { ClaimCommonTypes } from './types'
 import { ClaimStatus } from 'state/claim/actions'
 import { AMOUNT_PRECISION } from 'constants/index'
+import { useTokenBalance } from '@src/state/wallet/hooks'
```
Nitpick, can we remove `@src` here

elena-zh commented:
@alfetopito , this might be not related to the current PR, but I noticed that when I leave a PR tab opened for several hours, then get back to it, a lot is infura errors appear in the console. See the video: https://watch.screencastify.com/v/TJzMWMd7fJ09paKXSoD8
Could you please clarify, is this OK?

Commit:
Show network name on already claimed banner

Commit:
Do not show inner body msg when already claimed

Commit:
Show connect button on already claimed if disconnected

Commit:
Fix `@src` import

alfetopito commented:
@elena-zh addressed your suggestions:

## Connected
![Screen Shot 2022-02-10 at 09 18 57](https://user-images.githubusercontent.com/43217/153461715-2ad7ed27-74ef-4ac6-ac66-159c1cc09841.png)

## Not connected
![Screen Shot 2022-02-10 at 09 19 15](https://user-images.githubusercontent.com/43217/153461729-abdb763b-ad96-4b8c-810a-d0487c31e38f.png)

elena-zh commented:
Great! Thank you!
